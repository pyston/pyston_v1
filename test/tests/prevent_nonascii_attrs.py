def f(a):
    print a

try:
    f(**{u'a\u0180':3})
except TypeError as e:
    print e

try:
    setattr(object(), u"\u0180", None)
except TypeError as e:
    print e

try:
    hasattr(object(), u"\u0180")
except TypeError as e:
    print e

