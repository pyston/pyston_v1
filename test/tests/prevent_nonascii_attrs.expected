Cannot use non-ascii unicode strings as attribute names or keywords
Cannot use non-ascii unicode strings as attribute names or keywords
Cannot use non-ascii unicode strings as attribute names or keywords
